### ISSUE TITLE
HM310P software doesnt work on my windos11 system

### ISSUE THREAD

**Issue by georgejansen** [hobbyquaker/hanmatek-hm310p#7]
Wathever I do, I can't get the hm310p software working. After installation I only get the fantastic picture of the HM310P on my screen saying the Port is not open.. 
When click setting and choose com5 ( the port which shows in my device manager) there is no respons and I can't even close the Port setting screen; no reaction of the close button wath so ever.
Please help me !
Kind regards,
George

**Comment by Flughafenretter** [comment]
Have you already found a solution? I have the same problem.

**Comment by georgejansen** [comment]
No, i never recieved a reaction or a solution on this; So , it still doesnt
work yet

Op di 12 sep. 2023 13:48 schreef Flughafenretter ***@***.***>:
